src/resample_geotiff.py

```diff
@@ -70,7 +70,7 @@ def resample_geotiff(geotiff, width, outFormat, outFile, use_nn = False):
   if outFormat.upper() == 'GEOTIFF':
     gdal.Translate(outFile, raster, resampleAlg=resampleMethod, width=width)
   elif outFormat.upper() == 'JPEG' or outFormat.upper() == 'JPG':
-    if colorTable == None:
+    if colorTable is not None:
```

Comment:
We should be checking for colorTable is None here.

Reply:
Ack! That was sloppy of me -- fixed now